asset-transfer-basic/application-typescript-hsm/src/app.ts

```diff
@@ -0,0 +1,276 @@
+/*
```

Comment:
Can I recommend using https://github.com/evanw/node-source-map-support#readme ?
I've used this a lot since it was suggested to me... means that all the exception stack traces report the TypeScript file line number, not the JavaScript... little thing but *soo* useful.

Reply:
This is definitely a useful thing to know about. I'd rather not add it to the sample code because I want to keep the sample code using and depending on what is essential rather than nice to have to keep the focus, however I will mention this in the readme for users to investigate.